api/v1beta1/aerospikecluster_mutating_webhook.go

```diff
@@ -189,23 +190,29 @@ func (c *AerospikeCluster) updateRacksPodSpecFromGlobal(asLog logr.Logger) error
 			rack.PodSpec = *rack.InputPodSpec
 		}
 	}
-	return nil
 }
 
 func (c *AerospikeCluster) updateRacksAerospikeConfigFromGlobal(asLog logr.Logger) error {
-	for i, rack := range c.Spec.RackConfig.Racks {
-		var m map[string]interface{}
-		var err error
+	for i := range c.Spec.RackConfig.Racks {
+		rack := c.Spec.RackConfig.Racks[i]
```

Comment:
Reason why we have done this is because we were copying n number of bytes unnecessary. Is it really helping in that. Lint issue is fix but we are still creating a rack var of size n bytes.